### DISCUSSION TITLE
Hauppauge HDPVR Rocket  - anyone used one with a Windows SageTV installation?

### DISCUSSION THREAD

**Discussion by Sagetvmaster** [OpenSageTV/Community#157] [Capture Devices]
I'm looking at building what will likely be my FINAL SageTV server, based around a micro-PC concept with external storage.
For a tuner, which will be getting video from a cable, set-top box, has anyone had any experience with the smallish Hauppauge HDPVR Rocket?
I have spare HDPVR 1's kicking around, and could use those, but this just looked like it used a smaller amount of real estate.
Thanks

**Comment by JustFred51** [comment]
Looking at the Hauppauge website for the Rocket, it appears that it's still not going to work with SageTv.  There's no DirectShow support, which was the issue when the device was first introduced in 2013.  See: https://forums.sagetv.com/forums/showthread_t_60898.html?t=60898

For a capture device with a small footprint, it's hard to beat one of the URay HDMI network encoder boxes.  See: https://www.amazon.com/URayTech-Encoder-Protocols-Facebook-Platforms/dp/B07D78L3SZ.  I've purchased several of these from AliExpress for as low as $118 each.  Rock solid; way more reliable than the 2 HD-PVRs that they replaced.  Side benefit:  the URay boxes have HDMI inputs that automatically strip the HDCP nonsense, so no need to use an HDCP stripper between the STB and the capture box.

**Reply by Sagetvmaster** [reply]
Oh, I missed that.  Thanks for that info about DirectShow.  I'll look at this new kit you're using - seems promising.  My HD-PVR 1 has been working fine for 15 years now?  Never tried a HDPVR 2 because they didn't support 5.1 audio.

**Reply by pendragonsound** [reply]
I looked into the URay or a similar unit (all likely based on the same chips) several months ago. While it could encode into all sorts of audio formats, it only advertised itself as stereo PCM-capable over HDMI. One can read in-between the lines in the specs, because they never say what forms of bitstream audio the unit can accept. Just a heads-up if you want 5.1 audio.

**Reply by Sagetvmaster** [reply]
Another good bit of info.  Yes, I find 5.1 audio is more important than 4k.  Alot will disagree, but I come from a sound background.  Nothing beats proper surround

**Reply by wayner9** [reply]
But what content do you find that is properly encoded in 5.1?  Are TV shows?  Sports?

**Reply by emveepee** [reply]
My HDPVR captures from my Rogers XiOne will often be 5:1 but I don't know how you define "properly encoded". The raw ATSC streams OTA will probably be truer.

**Reply by wayner9** [reply]
I mean do the shows even have 5.1 audio in the first place.  Or maybe just stereo split across 5 channels, if they even have 5.1.  So does having 5.1 add anything to the experience, other than with movies.  

I seem to remember that the 5.1 audio was better back more than a decade in the early days of HD channels.
